core/chains/evm/headtracker/head_listener_test.go

```diff
@@ -156,6 +156,7 @@ func Test_HeadListener_SubscriptionErr(t *testing.T) {
 	}
 
 	for _, test := range tests {
+		test := test
```

Comment:
These don't run in parallel, but a closure captures a reference and then must have run late, during a later iteration. (could be an orphaned goroutine here too... :thinking: )